aws_lambda_builders/workflows/nodejs_npm/actions.py

```diff
@@ -0,0 +1,85 @@
+"""
+Action to resolve NodeJS dependencies using NPM
+"""
+
+import logging
+from aws_lambda_builders.actions import BaseAction, Purpose, ActionFailedError
+from .utils import OSUtils
+from .npm import SubprocessNpm, NpmExecutionError
+
+LOG = logging.getLogger(__name__)
+
+
+class NodejsNpmPackAction(BaseAction):
+
+    NAME = 'NpmPack'
+    DESCRIPTION = "Packaging source using NPM"
+    PURPOSE = Purpose.COMPILE_SOURCE
```

Comment:
is this really Compile Source? I think this is `COPY_SOURCE`. Compile is for things like golang which actually does the compilation

Reply:
ok, fixed in https://github.com/gojko/aws-lambda-builders/commit/2542cdbbcbaf4afe7ce17b1749a75804b96b1848